pkgs/development/python-modules/rectangle-packer/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  setuptools,
+  cython,
+}:
+
+buildPythonPackage rec {
+  pname = "rectangle-packer";
+  version = "2.0.2";
+
+  src = fetchFromGitHub {
+    owner = "Penlect";
+    repo = pname;
+    rev = version;
+    hash = "sha256-YsMLB9jfAC5yB8TnlY9j6ybXM2ILireOgQ8m8wYo4ts=";
+  };
+
+  nativeBuildInputs = [
+    cython
+    setuptools
+  ];
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail 'Cython<3.0.0' 'Cython'
+  '';
+
+  pythonImportsCheck = [ "rpack" ];
+
```

Comment:
Please enable tests.

Reply:
I'm unable to make the tests run. They fail to find the built modules, but if I import the package it works as expected as do packages using it.